lib/matplotlib/ticker.py

```diff
@@ -18,6 +18,9 @@
 `MultipleLocator`. It is initialized with a base, e.g., 10, and it picks
 axis limits and ticks that are multiples of that base.
 
+Locators must return at least a single tick below ``vmin`` and a single tick
+above ``vmax``.
```

Comment:
Seems like this should be changed to >=/<=, cf https://github.com/matplotlib/matplotlib/issues/25146#issuecomment-1431964579